assets/dev/js/editor/elements/models/base-model.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * The class propose is to share the common methods and properties between all available models.
+ */
+export default class BaseModel extends Backbone.Model {
```

Comment:
// Todo:  Extract each child condition to the relevant model (e.g. container conditions to container model)